arm-recoveryservicesbackup/readme.md

```diff
@@ -33,4 +35,15 @@ directive:
 csharp:
   namespace: Microsoft.Azure.Management.RecoveryServices.Backup
   output-folder: Generated/CSharp
+```
+
+### Python
+
+```yaml
+python:
+  namespace: azure.mgmt.recoveryservices.backup
+  package-name: azure-mgmt-recoveryservicesbackup
+  package-version: 1.0.0-preview
```

Comment:
Should be 0.1.0 or 1.0.0a1 to respect Python syntax:
https://www.python.org/dev/peps/pep-0440/#pre-releases